packages/contracts-bedrock/contracts/test/CommonTest.t.sol

```diff
@@ -134,7 +134,8 @@ contract L2OutputOracle_Initializer is CommonTest {
             startingBlockNumber,
             startingTimestamp,
             proposer,
-            owner
+            owner,
```

Comment:
slight nit: move towards named function args syntax, no need to do it in this PR if you don't want to but we should be trying to slowly move that direction when time permits :)